pysd/py_backend/functions.py

```diff
@@ -1154,9 +1154,33 @@ def active_initial(time, expr, init_val):
     else:
         return expr()
 
+def random_0_1():
+    """
+    Implements Vensim's RANDOM 0 1 function.
+
+    Returns
+    -------
+    A random number from the uniform distribution between 0 and 1.
+    """
+    return np.random.uniform(0,1)
 
 def random_uniform(m, x, s):
-    # TODO document
+    """
+    Implements Vensim's RANDOM UNIFORM function.
+
+    Parameters
+    ----------
+    m: int
+        Minimum value that the function will return.
+    x: int
+        Maximun value that the function will return.
+    s: int
```

Comment:
If we aren't going to implement the `s` functionality, should probably throw a warning if a nonzero value is used.

Reply:
Good idea. I add it now in a new commit.